scuttlebutt-test/Cargo.toml

```diff
@@ -21,4 +22,4 @@ tracing-subscriber = "0.3"
 assert_cmd = "2"
 predicates = "2"
 reqwest = { version = "0.11", default-features=false, features = ["blocking", "json"] }
-
+serial_test = "0.5.1"
```

Comment:
Let's implement tests that can be run in parallel using random ports. Two tests can run parallel using random ports in different ranges.

Reply:
Not sure if this works here and universal this trick is, but when listenign to port 0, linux picks a random UDP port that is available.